Runtimes/Core/core/CMakeLists.txt

```diff
@@ -339,6 +339,8 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/Swift.swiftmodule"
 emit_swift_interface(swiftCore)
 install_swift_interface(swiftCore)
 
+include("${SwiftCore_VENDOR_MODULE_DIR}/swiftCore.cmake" OPTIONAL)
```

Comment:
what do you think of naming this `VendorCustomizations.cmake`?

Reply:
Eventually we will have `_Concurrency` and `Onosupport` libraries built in this project, and we should be able to target changes at those libraries too.
We could put one toward the end of the main `CMakeLists.txt` and use that to target various libraries directly, I suppose?

Reply:
Oh, wait, I see `SwiftCore_VENDOR_MODULE_DIR` ... so this seems pretty reasonable.